src/Alpha_complex/include/gudhi/Alpha_complex/Alpha_kernel_d.h

```diff
@@ -0,0 +1,141 @@
+/*    This file is part of the Gudhi Library - https://gudhi.inria.fr/ - which is released under MIT.
+ *    See file LICENSE or go to https://gudhi.inria.fr/licensing/ for full license details.
+ *    Author(s):       Vincent Rouvreau
+ *
+ *    Copyright (C) 2020 Inria
+ *
+ *    Modification(s):
+ *      - YYYY/MM Author: Description of the modification
+ */
+
+#ifndef ALPHA_COMPLEX_ALPHA_KERNEL_D_H_
+#define ALPHA_COMPLEX_ALPHA_KERNEL_D_H_
+
+#include <CGAL/version.h>  // for CGAL_VERSION_NR
+
+#include <Eigen/Core>  // for EIGEN_VERSION_AT_LEAST
+
+#include <utility>  // for std::make_pair
+
+// Make compilation fail - required for external projects - https://github.com/GUDHI/gudhi-devel/issues/10
+#if CGAL_VERSION_NR < 1041101000
+# error Alpha_complex is only available for CGAL >= 4.11
+#endif
+
+#if !EIGEN_VERSION_AT_LEAST(3,1,0)
+# error Alpha_complex is only available for Eigen3 >= 3.1.0 installed with CGAL
+#endif
+
+namespace Gudhi {
+
+namespace alpha_complex {
+
+/**
+ * \class Alpha_kernel_d
+ * \brief Alpha complex kernel container.
+ * 
+ * \details
+ * The Alpha complex kernel container stores CGAL Kernel and dispatch basics computations in function of the weighted
+ * or not version of the Alpha complex.
+ */
+template < typename Kernel, bool Weighted = false >
+class Alpha_kernel_d {
+};
+
+// Unweighted Kernel_d version
+template < typename Kernel >
+class Alpha_kernel_d<Kernel, false> {
+ private:
+  // Kernel for functions access.
+  Kernel kernel_;
```

Comment:
Normally, there should be a single instance of the traits class (aka kernel), not one in triangulation + one in Alpha_kernel_d + etc. Or at least one should be a copy of the other. It doesn't matter for Epick_d/Epeck_d though, so not a priority.

Reply:
Maybe I should rename the variable `kernel_` in Alpha_complex.h as it seems confusing...
`kernel_` in Alpha_complex.h is of type `Alpha_kernel_d<Kernel, Weighted>`
`kernel_` in Alpha_kernel_d.h is of type `Kernel` and is where the 'reel' cgal kernel is stored.
Are you ok, if I rename `kernel_` in Alpha_complex.h as `alpha_kernel_d_` ?

Reply:
My comment was not about the name, it was about the fact that the member `kernel_` in Alpha_kernel_d is unnecessary, we can use `triangulation_->geom_traits()` to access the instance that already exists there. Having 2 independent instances is ok for Epick_d / Epeck_d, but it would cause problems for a kernel that stores information.